docker/collector/kafka/install.sh

```diff
@@ -41,4 +41,10 @@ EOF
 
 mkdir /kafka/logs
 
+echo "*** Cleaning Up"
+apk del jq curl --purge
+# TODO: eventually cleanup irrelevant binaries from RocksDB
+# https://issues.apache.org/jira/browse/KAFKA-10380
```

Comment:
biggest single jar is the embedded RocksDB with several OS binaries inside: 15MB